### PR TITLE
updated weather skill to answer most queries

### PR DESCRIPTION
**Pull request by AliAyub007**
Fixes issue #122 

Changes:
Added skill to answer different queries related weather in different way.

Screenshots for the change: 

weather1

weather2

Etherpad Link: http://dream.susi.ai/p/updatedweather

### CODE REVIEW

**Review by hpdang — CHANGES_REQUESTED** [review]
I wonder if we can use the specific weather API instead taking wolframalpha for everything. the openweathermap.org doesn't work? Accuweather or Weatherunderground (wunderground) also offer free API for developers

**Comment by AliAyub007** [comment]
@hpdang openweather was not giving accurate responses and weatherunderground has limited number of api calls i.e 500 API calls per day and so do other APIs. If I will make a skill with it now it will work fine but as soon we get more number of users our skill will go down and we might have to pay for APIs which is not a best way.

**Comment by hpdang** [comment]
@AliAyub007 please ask fellow developers to review your PR. Don't forget our peer review practice. Thank you.

**Comment by AliAyub007** [comment]
@Manveer695 @saurabhjn76 @dynamitechetan @DravitLochan   Kindly review this PR Thank you.

File: models/general/knowledge/en/weather_information.txt [hunk 1]
```diff
@@ -1,60 +1,161 @@
...
+I hate summers too I think we are alike.
+
+Why does it rain|why it rains
+Because clouds get filled with water.
+
+I love snowing|I love snow|I love snow falling|I love when it snows|I love when it is snowing|I like snow|I like when it snows|I like when it is snowing
+I love snow too but I am sad I can't throw snow balls
+
+I hate snowing|I hate snow|I hate snow falling|I hate when it snows|I hate when it is snowing|I don't like snow|I do not like snow|I don't like when it snows|I do not like when it snows|I don't like when it is snowing|I do not like when it is snowing
+I love snow too but I am sad I can't throw snow balls
```
Comment (saurabhjn76) [inline models/general/knowledge/en/weather_information.txt]: I think answer should be different in this case or "too" can be removed

File: models/general/knowledge/en/weather_information.txt [hunk 2]
```diff
@@ -1,60 +1,161 @@
...
-# we need that for the weather api
-kelvin to celsius *
+mentions * *
+!console:The following users are mentioned with $1$: @$k0$ ($v0$ times), @$k1$ ($v1$ times), @$k2$ ($v2$ times)
+{
+"url":"http://api.loklak.org/api/search.json?q=$2$&source=cache&count=0&fields=mentions&limit=6",
+"path":"$.aggregations.mentions"
+}
+eol
+
```
Comment (saurabhjn76) [inline models/general/knowledge/en/weather_information.txt]: Above skills do not give weather information, maybe we should change it to some other file?

File: models/general/knowledge/en/weather_information.txt [hunk 3]
```diff
@@ -1,60 +1,161 @@
...
...ining tomorrow |Are there any chances of rain tomorrow |can it rain tomorrow |will it rain tomorrow |will there be rain tomorrow|Will it rain today |Will it be raining|Are there any chances of rain today |can it rain today|will it rain today|will there be rain today|Will it rain|Will it be raining |Are there any chances of rain |can it rain |will it rain|will there be rain |will it be raining today |Are there any chances of rain|Will there be clouds|Will it be cloudy |will it be cloudy weather|Is it cloudy|Is it cloudy|Is cloudy weather |Is weather cloudy |Will there be clouds today |Will it be cloudy today |will it be cloudy weather today ||Is it cloudy today |Is weather cloudy today |Will there be hail|Will there be hail today |Will it hail |will it hail today |Will it be hailing today |Will it be hailing |will it be hailing weather |will it be hailing weather today |Is it hailing |Is it hailing today |Is weather clear |Is weather clear today |Will there be snow |Will there be snow today |Will there be snow |Will it snow |will it snow today |Will it be snowing today |Will it be snowing |will it be snowing weather today |will it be snowing weather|Is it snowing today Is it snowing
```
Comment (saurabhjn76) [inline models/general/knowledge/en/weather_information.txt]: Is it snowing today Is it snowing missing optional symbol (|)

File: models/general/knowledge/en/weather_information.txt [hunk 4]
```diff
@@ -1,60 +1,144 @@
...
-"path":"$.list[9].weather[0]"
+"url":"https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20weather.forecast%20where%20woeid%20in%20(select%20woeid%20from%20geo.places(1)%20where%20text%3D%22$1$%2C%20ak%22)&format=json&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys",
+"path":"$.query.results.channel.astronomy"
 }
-eol
\ No newline at end of file
+eol
+
+what is sunrise time in *|Sunrise time in *|sunrise in *|sun rise in *|Tell me about sunrise time|what is the sunrise time in *|what is sun rise time in *|Sun rise time in *|Tell me about sun rise time|what is the sun rise time in *
+!console:Sunset time in $1$ is $sunrise$
```
Comment (Manveer695) [inline models/general/knowledge/en/weather_information.txt]: I think here it should be sunrise.

**Review by Manveer695 — APPROVED** [review]

**Pull request merged**